libs/blockscout-db/Cargo.toml

```
@@ -6,5 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-migration = { path = "./migration" }
-entity = { path = "./entity" }
+blockscout-db-migration = { path = "./blockscout-db-migration" }
+blockscout-db-entity = { path = "./blockscout-db-entity" }
```

I suggest to left paths the same, and update only the crate name:
```suggestion
blockscout-db-migration = { path = "./migration" }
blockscout-db-entity = { path = "./entity" }
```

That way we will avoid problems with the same crate names, while will not create an unnecessary prefixes

---

Agree, I think we should only change crate name